cmd/docker/docker.go

```diff
@@ -32,14 +31,36 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 		SilenceUsage:     true,
 		SilenceErrors:    true,
 		TraverseChildren: true,
-		Args:             noArgs,
+		Args: func(_ *cobra.Command, _ []string) error {
```

Comment:
Isn't this the same as no function provided ?

Reply:
Yes, I wasn't quite sure yet if I did actually need to do something here (relating to the issue with `--help` I mentioned in the intro). Will verify all that and drop if I can.